GPU/GPUTracking/utils/.gitattributes

```diff
@@ -1 +0,0 @@
-/callvc.bat binary
```

Comment:
@syano0822 also this file seems to be unrelated to your PR, could you please restore it?